example/test/unmount_test.dart

```diff
@@ -23,7 +23,7 @@ void main() {
   print('What');
   final mountedNode = querySelector('#content');
 
-  querySelector('#mount').onClick.listen((_) => react_dom.render(simpleComponent({}), mountedNode));
+  querySelector('#mount')!.onClick.listen((_) => react_dom.render(simpleComponent({}), mountedNode));
```

Comment:
Is there a reason this couldn't just be 

```dart
querySelector('#mount')?.onClick?.listen(//...
```

Reply:
Since we always expect the node to be there, I think it throwing if this is null is better behavior than it silently failing.